styles/components/card.stories.js

```diff
@@ -0,0 +1,22 @@
+export default {
+  title: "Components/card",
```

Comment:
May you write card with a Capital C to follow the Convention

Reply:
yes thanks you are right